frontend/src/pages/Overview.js

```diff
@@ -604,3 +611,60 @@ const getAccountNumber = (account) => {
       return 0;
   }
 };
+
+
+function secondsToTime(seconds) {
```

Comment:
You are adding seconds to the current time and returning the formatted date. In that case, you can do it by moment with a simple line. 
 moment().add(100, 'seconds').format('MM/DD/YYYY HH:mm:ss A')